src/A2C_model.py~

```diff
@@ -40,22 +45,27 @@ class Model(Model):
         i = self.batchnormalization1(i)
         i = self.conv1(i)
         #i = self.maxpool2(i)
+        i = self.batchnormalization2(i)
         i = self.conv2(i)
         #i = self.maxpool3(i)
+        i = self.batchnormalization3(i)
         i = self.conv3(i)
         flattened = self.flatten(i)
+
         # actor dense layers
         x = self.d1(flattened)
-        x = self.batchnormalization2(x)
         x = self.d2(x)
-        #x = self.dropout(x)
+        x = self.batchnormalization4(x)
         x = self.d3(x)
+        #x = self.dropout2(x)
```

Comment:
It might be okay for now as you're still working through it, but in the final version of your code don't include commented out lines like this